src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends FigCol {
```

Comment:
Don't put all behavior into a single interface if the methods are conceptually different from each other. If 'FigCol' contains methods like 'draw()' and 'getArea()', consider splitting it into separate interfaces, as these methods are not conceptually close to each other.